marklogic-data-hub-central/ui/e2e/cypress/e2e/persistence/persistence.cy.tsx

```diff
@@ -399,21 +399,21 @@ describe("Validate persistence across Hub Central", () => {
     cy.waitForAsyncRequest();
     browsePage.waitForSpinnerToDisappear();
     browsePage.databaseSwitch("final");
-    graphExplore.getSearchBar().type("Adams");
-    browsePage.getApplyFacetsButton().click();
+    graphExplore.getSearchBar().type("Bowman");
+    browsePage.getApplyFacetsButton().should("be.visible", {timeout: 3000}).click();
```

Comment:
Suggestion: If we need to apply the same logic to a method, it is better to include it in getApplyFacetsButton() than having to do it in all the implementations of the method.